src/main/webapp/WEB-INF/tags/instructor/results/filterEditButton.tag

```diff
@@ -0,0 +1,21 @@
+<%@ tag description="instructorFeedbackResultsTop - Filter Panel Edit Button" %>
```

Comment:
Also, `filterEditButton` is a misleading name, as this is actually the expand/collapse button.